spec/features/authorization_editor_spec.rb

```diff
@@ -1,16 +1,13 @@
 require 'rails_helper'
-      #save_and_open_page
+
 RSpec.describe "a user" do
+  before(:each) do
+    @person = create(:person)
+  end
+
   describe "in the reader role" do
-    before (:each) do
-      @person = create(:person)
-      somebody = create(:user)
-      somebody.roles << create(:role, name: 'Reader')
-      visit new_user_session_path
-      fill_in 'user_email', :with => somebody.email
-      fill_in 'user_password', :with => somebody.password
-      click_on 'Sign in'
-    end
+    before (:each) { sign_in_as('Reader') }
```

Comment:
In general, I think so, though in this case they are nested, since for both the `reader` tests and the `editor` tests we want a `person`, but we want to be signed in as a different role for each one.  Some documentation [here](http://www.relishapp.com/rspec/rspec-core/docs/hooks/before-and-after-hooks#%60before%60/%60after%60-context-blocks-are-run-once).